daemon/redis.c

```diff
@@ -812,7 +726,6 @@ static int json_get_hash(struct redis_hash *out, struct call* c,
 	out->ht = g_hash_table_new_full(g_str_hash, g_str_equal, free, freeReplyObject);
```

Comment:
Ok thanks for the comments. the "single malloc" I understood that you like to do it, if not just drop a quick message.